components/application-mgt/org.wso2.carbon.device.application.mgt.core/src/main/java/org/wso2/carbon/device/application/mgt/core/dao/ApplicationDAO.java

```diff
@@ -66,4 +59,13 @@ void changeLifecycle(String applicationUUID, String lifecycleIdentifier, String
     void updateScreenShotCount(String applicationUUID, int tenantId, int count) throws
             ApplicationManagementDAOException;
 
+    Category addCategory(Category category) throws ApplicationManagementDAOException;
```

Comment:
I think, we have forgotten to add the method comments here, can you please add the comments?